rcswitch/user_doc.rst

```diff
@@ -0,0 +1,134 @@
+.. index:: Plugins; rcswitch (Brennenstuhl RCS 1000 N)
+.. index:: rcswitch
+
+================
+Plugins/rcswitch
+================
+
+
+Anforderungen
+-------------
+Das rcswitch Plugin beötigt folgenden Anforderungen an Hanrdware- und Software-Komponenten.
```

Comment:
Da das Plugin keine Erweiterung (bzw. Plugin-In Replacement) für das alte rcswitch Plugin ist, sollte hier

```
Das rcswitch2 Plugin benötigt ...
```
stehen. (Außerdem würde ich noch ein 'n' spendieren 😉

Reply:
erl. in PR für `rcs1000n`